weather.css

```diff
@@ -0,0 +1,54 @@
+
+#font
+ {
+    color: white;
+    font-size: 100px;
+    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  }
+  table{
+    border-collapse: collapse;
+    padding: 20px;
+    color: white;
+    font-size: 65px;
+    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+
+  }
+  
+
+  #slika{
```

Comment:
Using ID selectors is generally discouraged for styling purposes, as they have high specificity and can lead to inflexible styles. Please use classes